application/src/main/java/org/thingsboard/server/install/ThingsboardInstallService.java

```diff
@@ -284,6 +293,7 @@ public void performInstall() {
                 systemDataLoaderService.createQueues();
 //                systemDataLoaderService.loadSystemPlugins();
 //                systemDataLoaderService.loadSystemRules();
+                installScripts.migrateLwm2mResources();
```

Comment:
better to use systemDataLoaderService for loading lwm2m resources
`systemDataLoaderService.loadSystemResources();`

Reply:
OK